webapp/.babelrc

```diff
@@ -1 +1,4 @@
-{ "presets": ["es2015", "react", "stage-0"] }
+{ 
+    "presets": ["es2015", "react", "stage-0", "airbnb"],
```

Comment:
You get an AirBnB voucher with each new deployment ;)
The tests were failing for new ES6 code and I wanted to bring this inline with the testing approach in create-react-app. That uses enzyme for testing, which is an airbnb  project.

Babel is a pre-processor that converts ES6 style code into native ES5 Javascript. This is so that projects can use ES6 now without waiting years for browsers to handle ES6 natively. There are processors for react (converting JSX) and airbnb is doing some magic... it fixed the issues that I had and allowed enzyme tests to work.